### DISCUSSION TITLE
How are ambiguous alleles matched?

### DISCUSSION THREAD

**Discussion by adams-cameron** [PGScatalog/pgsc_calc#221] [Q&A] [answered]
Could you provide more detail about the strategies used for matching variants with ambiguous alleles. 
Thanks!

**Comment by smlmbrt** [comment, marked as answer]
Thanks for your question, currently we just match the alleles as-is (basically assuming the scoring file and target genomes are the same orientation). By default we drop ambiguous variants from the scoring files, as being the same orientation is likely too strong an assumption (although may be correct when they do match). You can see some of the options to adjust the behaviour of the matching here: https://pgsc-calc.readthedocs.io/en/latest/reference/params.html#matching-options.

We have considered doing allele-frequency based matching/harmonization; however, we didn't implement it because a minority of scoring files in the PGS Catalog contain effect allele frequencies.
